plugin/src/main/scala/com/typesafe/genjavadoc/AST.scala

```diff
@@ -93,9 +108,27 @@ trait AST { this: TransformCake =>
     }
   }
 
-  case class MethodInfo(access: String, pattern: String => String, ret: String, name: String, comment: Seq[String]) extends Templ {
-    def sig = pattern(s"$ret $name")
+  case class MethodInfo(access: String, pattern: String => String, ret: String, name: String, comment: Seq[String], d: Option[DefDef] = None) extends Templ {
```

Comment:
Might have been nice to only include the annotations here rather than the full `DefDef`?